kuma/javascript/src/document-provider.jsx

```diff
@@ -80,9 +80,20 @@ export default function DocumentProvider(
                     }
                 })
                 .then(json => {
-                    window.scrollTo(0, 0);
-                    setDocumentData(json);
-                    body.style.opacity = '1';
+                    if (json.redirectURL) {
+                        window.location = json.redirectURL;
```

Comment:
I'd add a comment before this line:

// We got a redirect to a document that can't be handled
// via the /api/v1/doc/ API, so we just do a full page load of that document.